corehq/apps/users/validation.py

```diff
@@ -59,3 +61,16 @@ def validate_profile_required(profile_name, domain):
         raise ValidationError(
             _("A profile assignment is required for Mobile Workers.")
         )
+
+
+def validate_assigned_locations_has_users(domain, assigned_location_ids):
+    error_message_location_not_has_users = _("These locations cannot have users assigned because of their "
```

Comment:
Shouldn't we remove `error_message_location_not_has_users` from `LocationValidator` class?

Reply:
yes you're right, thanks
[d3b2d3a](https://github.com/dimagi/commcare-hq/pull/35448/commits/d3b2d3a38f9574d0bf65625d547c341cd6b4b88f)